modules/nf-core/crabz/compress/tests/main.nf.test

```diff
@@ -0,0 +1,61 @@
+nextflow_process {
+
+    name "Test Process CRABZ_COMPRESS"
+    script "../main.nf"
+    process "CRABZ_COMPRESS"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "crabz"
+    tag "crabz/compress"
+
+    test("sarscov2 - genome - fasta") {
+
+
+        when {
+            process {
+                """
+
+                input[0] = [
+                    [ id:'test'], // meta map
+                    file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
```

Comment:
You can should the path to the other spec which was also recommended to me so that it looks like this
`file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/fastq/test_1.fastq.gz', checkIfExists: true)`

Reply:
Is this in the documentation? I used `nf-core modules create`.